posthog/batch_exports/service.py

```diff
@@ -249,6 +253,36 @@ def pause_batch_export(temporal: Client, batch_export_id: str, note: str | None
     batch_export.last_paused_at = dt.datetime.now(dt.timezone.utc)
     batch_export.save()
 
+    return True
+
+
+async def apause_batch_export(temporal: Client, batch_export_id: str, note: str | None = None) -> bool:
+    """Pause this BatchExport.
+
+    We pass the call to the underlying Temporal Schedule.
+
+    Returns:
+        `True` if the batch export was paused, `False` if it was already paused.
+    """
+    try:
+        batch_export = await BatchExport.objects.aget(id=batch_export_id)  # type: ignore
```

Comment:
These type ignore comments are necessary because our version of django-stubs is 3 years old and doesn't have the async methods defined. Trying to bump the version leads to a lot of errors from mypy.